make uninstall fails to complete - tries to cd to non-existent dir libr/sysregs/d

### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Ubuntu x86 64
| File format of the file you reverse (mandatory)      | n/a
| Architecture/bits of the file (mandatory)            |  n/a
| r2 -v full output, **not truncated** (mandatory)     | radare2 2.4.0-git 17386 @ linux-x86-64 git.2.2.0-575-gfb0d79e6a commit: fb0d79e6ab14d90377602ef1d509228d2b6bc28e build: 2018-02-23__22:51:46


### Expected behavior

make uninstall completes without errors

### Actual behavior

make uninstall does not fully complete due to error generated by cd to directory that doesn't exist (line 281) and as a result 'make uninstall' is never run in dir `libr/anal/d` (line 282). 

### Steps to reproduce the behavior 

1. Clone radare2 GitHub repo
2. From root dir of radare2 repo, run `sys/install.sh` Provide sudo password when prompted
3. From root dir of radare2 repo attempt: `cd libr/sysregs/d` to confirm it is not there post install.
4. From root dir of radare2 repo, run `sudo make uninstall`
5. Observe the error message:

```
/bin/sh: 1: cd: can't cd to libr/sysregs/d
Makefile:275: recipe for target 'uninstall' failed
make: *** [uninstall] Error 2
```


### Additional Logs, screenshots, source-code,  configuration dump, ...

 Full uninstall output
```
pgrohe@zen:~/built_from_source/radare2$ sudo make uninstall
rm -f /usr/bin/r2-indent
rm -f /usr/bin/r2-docker
cd libr && make uninstall PARENT=1
make[1]: Entering directory '/home/pgrohe/built_from_source/radare2/libr'
# libraries
/usr/lib/libr_anal.so
/usr/lib/libr_asm.so
/usr/lib/libr_bin.so
/usr/lib/libr_bp.so
/usr/lib/libr_config.so
/usr/lib/libr_cons.so
/usr/lib/libr_core.so
/usr/lib/libr_crypto.so
/usr/lib/libr_debug.so
/usr/lib/libr_egg.so
/usr/lib/libr_flag.so
/usr/lib/libr_fs.so
/usr/lib/libr_hash.so
/usr/lib/libr_io.so
/usr/lib/libr_lang.so
/usr/lib/libr_magic.so
/usr/lib/libr_parse.so
/usr/lib/libr_reg.so
/usr/lib/libr_search.so
/usr/lib/libr_socket.so
/usr/lib/libr_syscall.so
/usr/lib/libr_util.so
# object archives
# includes
if [ -h "/usr/include/libr" ]; then \
	rm -f "/usr/include/libr" ; \
else \
	(cd include && for FILE in * ; do rm -f "/usr/include/libr/$FILE" ; done) ; \
fi
rm: cannot remove '/usr/include/libr/r_util': Is a directory
rm: cannot remove '/usr/include/libr/sdb': Is a directory
rm: cannot remove '/usr/include/libr/sflib': Is a directory
Makefile:134: recipe for target 'uninstall' failed
make[1]: [uninstall] Error 1 (ignored)
# programs
# plugins
/usr/lib/radare2/2.4.0-git/makeplugin.sh
rm -rf "/usr/lib/radare2/2.4.0-git"
rm -f "/usr/lib/pkgconfig/libr.pc"
rm -f "/usr/lib/pkgconfig/r_"*.pc
# test programs
rm -rf "/usr/bin/libr-test"
# TODO: use for FILE in LIBS (like in binr/Makefile)
rm -rf "/usr/include/libr"
rm -rf "/usr/lib/libr_*.so.2.4.0-git"
rm -rf "/usr/lib/libr_*.so.0"
rm -rf "/usr/lib/libr_*.so"
rm -rf "/usr/lib/libr_*.a"
rm -rf "/usr/lib/libr2.so"*
rm -rf "/usr/lib/libr2.a"
rm -rf "/usr/lib/libr.so"*
rm -rf "/usr/lib/libr.a"
rm -rf "/usr/share/doc/radare2"
libr aka radare2 has been uninstalled from PREFIX=/usr
make[1]: Leaving directory '/home/pgrohe/built_from_source/radare2/libr'
cd binr && make uninstall PARENT=1
make[1]: Entering directory '/home/pgrohe/built_from_source/radare2/binr'
cd .. ; \
for BINARY in rax2 rasm2 rabin2 rahash2 radiff2 radare2 rafind2 rarun2 ragg2 r2agent ; do rm -f "/usr/bin/$BINARY" ; done ; \
rm -f "/usr/bin/ragg2-cc" ; \
rm -f "/usr/bin/r2pm" ; \
rm -f "/usr/bin/r2"
make[1]: Leaving directory '/home/pgrohe/built_from_source/radare2/binr'
cd shlr && make uninstall PARENT=1
make[1]: Entering directory '/home/pgrohe/built_from_source/radare2/shlr'
rm -f /usr/lib/libr_shlr.a
make[1]: Leaving directory '/home/pgrohe/built_from_source/radare2/shlr'
cd libr/syscall/d && make uninstall PARENT=1
make[1]: Entering directory '/home/pgrohe/built_from_source/radare2/libr/syscall/d'
rm -rf "/usr/share/radare2/2.4.0-git/syscall"
make[1]: Leaving directory '/home/pgrohe/built_from_source/radare2/libr/syscall/d'
cd libr/sysregs/d && make uninstall PARENT=1
/bin/sh: 1: cd: can't cd to libr/sysregs/d
Makefile:275: recipe for target 'uninstall' failed
make: *** [uninstall] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -278,7 +278,6 @@ deinstall uninstall:
 	cd binr && ${MAKE} uninstall PARENT=1
 	cd shlr && ${MAKE} uninstall PARENT=1
 	cd libr/syscall/d && ${MAKE} uninstall PARENT=1
-	cd libr/sysregs/d && ${MAKE} uninstall PARENT=1
 	cd libr/anal/d && ${MAKE} uninstall PARENT=1
 	@echo
 	@echo "Run 'make purge' to also remove installed files from previous versions of r2"

